crates/sqllib/src/lib.rs

```diff
@@ -22,16 +24,19 @@ pub mod operators;
 pub mod source;
 #[doc(hidden)]
 pub mod string;
+#[doc(hidden)]
 pub mod timestamp;
 #[doc(hidden)]
 pub mod uuid;
+#[doc(hidden)]
 pub mod variant;
 
 pub use array::Array;
 pub use binary::ByteArray;
 #[doc(hidden)]
 pub use geopoint::GeoPoint;
 pub use interval::{LongInterval, ShortInterval};
+pub use map::Map;
```

Comment:
why is this necessary?

Reply:
We seem to re-export all sql types in the crate root which is fine I think.. the reason I ran into this is when trying to help some user with UDF code and I couldn't figure out what Map is when looking in docs.rs because it didn't show up.. so now you can find it in the docs